src/common/components/Header/Header.tsx

```diff
@@ -54,8 +65,19 @@ const SearchBar = () => {
         <input
           className='placeholder-gray-b0 h-41px bg-gray-f2 bg-opacity-50 rounded-full px-3 w-180px md:pl-46px md:pr-5 md:w-300px lg:w-300px xl:w-352px'
           placeholder='Search creator or NFT'
+          onKeyPress={onKey}
         />
       </div>
+
+      <SearchResults
+        open={openSearchResults}
+        onClose={toggleSearchResults}
+        foundNFTs={mockNFTs}
+        foundMembers={mockMembers}
+        onChangeTimeRange={() => {
+          /**/
```

Comment:
Can we remove this line?

Reply:
If I remove comment line, eslint will warn about empty func body.